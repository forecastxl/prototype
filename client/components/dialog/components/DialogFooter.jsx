import styled from 'styled-components'

const DialogFooter = styled.div`
  text-align: center;
  margin: 1.5rem 0 1rem;
`

export default DialogFooter
