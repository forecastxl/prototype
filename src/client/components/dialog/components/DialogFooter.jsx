import styled from 'styled-components'

const DialogFooter = styled.div`
  margin: 1.5rem 0 1rem;
  text-align: center;
`

export default DialogFooter
