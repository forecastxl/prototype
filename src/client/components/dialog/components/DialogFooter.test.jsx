/**
 * @jest-environment node
 */

import 'jest-styled-components'
import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import DialogFooter from './DialogFooter'

describe('<DialogFooter />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<DialogFooter />)
    expect(shallowToJson(wrapper)).toMatchStyledComponentsSnapshot()
  })
})
