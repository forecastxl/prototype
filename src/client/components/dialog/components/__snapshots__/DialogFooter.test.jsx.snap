// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DialogFooter /> renders correctly 1`] = `
.bbMsWb {
  margin: 1.5rem 0 1rem;
  -webkit-text-align: center;
  text-align: center;
}

<div
  className="sc-bdVaJa bbMsWb"
/>
`;
